WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -648,6 +653,49 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
         mediaInserterHelper.cancelUploadOf(media: media)
     }
 
+    func gutenbergDidRequestToSetFeaturedImage(for mediaID: Int32) {
+        // dismiss current controller in order to allow the new alert controller to be presented
+        presentedViewController?.dismiss(animated: false, completion: nil)
+
+        let featuredImageId = post.featuredImage?.mediaID
+        let mediaIdNoFeaturedImageSet = 0
+
+        if featuredImageId != nil && mediaID == featuredImageId as! Int32 {
+            // nothing special to do, trying to set the image that's already set as featured
+            return
+        }
```

Comment:
If the goal here is to exit the function if `featuredImageId` is equal to `mediaID`, then you could consider using Swift's guard-statement. It's often used to abort a function if certain conditions are not met:
```
        guard featuredImageId as? Int32 != mediaID else {
            return
        }
```

Reply:
Thank you for this! I've gone ahead to refactor the if statements to use guard in https://github.com/wordpress-mobile/WordPress-iOS/pull/16426/commits/25da33ea7dd83766704ec42ef6d067e554a86be0 and will know to consider using guard if I do any further iOS work in the future. :D